modules/rest-api/pages/rest-bucket-create.adoc

```diff
@@ -316,32 +317,67 @@ No object is returned.
+In Couchbase Server Community Edition, the default and only valid value is `couchstore`.
+For more information, see xref:learn:buckets-memory-and-storage/storage-engines.adoc[Storage Engines].
 
-NOTE: You can edit this value after initially creating the bucket. Couchbase Server sets the new backend value globally. 
+NOTE: You can edit this value after initially creating the bucket. 
+Couchbase Server sets the new backend value globally. 
 However, this change does not convert the bucket to the new backend storage engine. 
-Instead, Couchbase Server adds overrides to every node containing the bucket to indicate that their vBuckets are still in the old format. You must take additional steps to complete the migration to the new storage backend. 
+Instead, Couchbase Server adds overrides to every node containing the bucket to indicate that their vBuckets are still in the old format. 
+You must take additional steps to complete the migration to the new storage backend. 
 See xref:manage:manage-buckets/migrate-bucket.adoc[] for more information.
 
 [#example-storage-backend]
 ==== Example: Specifying the Storage Backend
 
-A minimum of 1024 MiB is required if the `magma` option is used; a minimum of 100 MiB if the default `couchstore` is used.
+The following example creates a new bucket, named `testBucket`, with the Magma storage backend.
 
 [source,bash]
 ----
 curl -v -X POST http://127.0.0.1:8091/pools/default/buckets \
 -u Administrator:password \
--d ramQuota=1024 \
+-d ramQuota=300 \
 -d storageBackend=magma \
 -d name=testBucket
 ----
 
 If successful, the call returns a `202 Accepted` notification.
 No object is returned.
 
+
+[#numvbuckets]
+=== numVbuckets
+
+Sets the number of vBuckets for a Magma bucket.
+The possible values are `128` or `1024`.
+If you do not supply this value (or if you supply a value other than `128` or `1024`), Couchbase Server uses the default value of `128`.
+
+If you set `storageBackend` to `couchstore`, the number of vBuckets is always 1024, and Couchbase Server ignores this parameter if you provide it.
+
+NOTE: You cannot change the number of vBuckets for a bucket after creating it.
+If you need a bucket with a different number of vBuckets, you must create a new bucket with the desired number of vBuckets and then migrate your data to it.
+See xref:manage:manage-buckets/migrate-bucket.adoc#xdcr-migration[XDCR Storage Backend Migration] for more information.
+
+[#example-numvbuckets-create]
+==== Example: Setting the Number of vBuckets
+
+The following example creates a new bucket named `testBucket` with the Magma stage backend assigns it `1024` vBuckets. 
+It also sets the `ramQuota` to `1024`, which is the minimum value allowed for a Magma bucket with 1024 vBuckets.
+
+[source,console]
+----
+curl -X POST http://127.0.0.1:8091/pools/default/buckets \
+     -u Administrator:password \
+     -d name=testBucket \
+     -d storageBackend=magma \
+     -d numVbuckets=1024 \
```

Comment:
The parameter name is `numVBuckets` -- it is very important that customers use the exact spelling, so we should make sure that whenever the REST API parameter name is mentioned, that it has the correct spelling.

If it does not have the correct spelling when used, because it's an optional parameter, it will be silently ignored (a 202 will be returned), so the customer may not be aware that anything is wrong.